demonstrations/tutorial_classical_kernels.py

```diff
@@ -0,0 +1,491 @@
+r"""
+
+.. _classical_kernels:
+
+Emulating classical kernels
+============================
+
+.. meta::
+    :property="og:description": Show how to approximate the Gaussian kernel.
+    :property="og:image": https://pennylane.ai/qml/_images/boring_kernel.jpg
+
+.. related::
+
+   tutorial_kernels_module Training and evaluating quantum kernels
+   tutorial_kernel_based_training Kernel-based training of quantum models with
+   scikit-learn
+   tutorial_expressivity_fourier_series Quantum models as Fourier series
+
+
+*Author: Elies Gil-Fuster (Xanadu resident). Posted: DD MMMM 2021.*
+
+In this demo we will briefly revisit some notions of kernel-based Machine
+Learning (ML) and introduce one very specific `Quantum Embedding Kernel (QEK)
+<https://pennylane.ai/qml/demos/tutorial_kernels_module.html>`_.
+We will also use this QEK to demonstrate we can approximate a large class of
```

Comment:
```suggestion
We will use this QEK to demonstrate we can approximate a large class of
```

It's  kind of the main thing...

Reply:
Maybe "important" class? Size is relative in function spaces...

Reply:
yes, important is definitely more accurate